oracle/controllers/inttest/usertest/user_test.go

```diff
@@ -21,6 +21,10 @@ import (
 	"testing"
 	"time"
 
+	commonv1alpha1 "github.com/GoogleCloudPlatform/elcarro-oracle-operator/common/api/v1alpha1"
```

Comment:
nit: can you keep these imports where they were? We try to separate the elcarro imports from other imports.

Reply:
Fixed